src/en/getting-started-general.md

```diff
@@ -25,8 +25,8 @@ Juju is currently available for Ubuntu, CentOS, MacOSX and Windows.
 
 ### Ubuntu
 
-To install Juju, you simply need to grab the 'juju' package from the
-PPA:
+To install Juju on Ubuntu 16.04 LTS (xenial), you simply need to grab the
```

Comment:
Apologies for pedantry, but I think we usually capitalise 'Xenial' (at least when we mention it like this - same with Trusty below).